style/style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: sans-serif;
+  color: #333;
+  line-height: 1.5;
+  height: 100vh;
+  background: linear-gradient(to right, #353a5f, #cacfe3);
+}
+
+h1 {
```

Comment:
@anndanielyan add class name, and then give a styles

Reply:
I know that every page must have only one h1. That's why I didn't add class, but now I will add it.

Reply:
If one day we will change that h1 tag to h2, you need to change it everywhere, but if you add class, its doesn't mean which tag you will change

Reply:
ah yes, I understood. Will I open a new branch for this?

Reply:
nope
just commit changes